### DISCUSSION TITLE
Logo TV

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#756] [Q&A]
### Channel Name

LOGOTV

### Channel ID

LogoEast.us

### Country

USA

### Language

English

### Website

http://www.logotv.com

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L34-L35
